applications/FluidDynamicsApplication/python_scripts/navier_stokes_solver_vmsmonolithic.py

```diff
@@ -286,17 +303,32 @@ def AddVariables(self):
         self.main_model_part.AddNodalSolutionStepVariable(KratosMultiphysics.NORMAL)
         self.main_model_part.AddNodalSolutionStepVariable(KratosMultiphysics.Y_WALL)
         self.main_model_part.AddNodalSolutionStepVariable(KratosCFD.Q_VALUE)
+
+        # Adding variables required for the turbulence modelling
+        if self.turbulence_model_solver:
```

Comment:
```suggestion
        if hasattr(self, '_turbulence_model_solver'):
```
By checking in this way you can avoid to save a  `self.turbulence_model_solver = None`. This can be extended to the rest of the solver.

Note that this also renames `self.turbulence_model_solver` variable to `self._turbulence_model_solver`. As far as I understand it is a private variable.

Reply:
done :)